e3sm_diags/derivations/formulas.py

```diff
@@ -77,15 +77,16 @@ def qflx_convert_to_lhflx_approxi(var: xr.DataArray):
 
 
 def pminuse_convert_units(var: xr.DataArray):
-    if (
-        var.attrs["units"] == "kg/m2/s"
-        or var.attrs["units"] == "kg m-2 s-1"
-        or var.attrs["units"] == "kg/s/m^2"
-    ):
-        # need to find a solution for units not included in udunits
-        # var = convert_units( var, 'kg/m2/s' )
-        var = var * 3600.0 * 24  # convert to mm/day
-        var.attrs["units"] = "mm/day"
+    if hasattr(var, "units"):
```

Comment:
Fixing issue 8 in https://github.com/E3SM-Project/e3sm_diags/pull/752#issuecomment-1901381465. xarray flag an error when dataarray doesn't have a units, while `cdms2` doesn't.

Reply:
I mean `cdms2` doesn't flag an error in this case.